config/logging.conf.sample

```diff
@@ -0,0 +1,44 @@
+[loggers]
```

Comment:
please add comments to each point in configuration. Comments helpful for someone who gonna create config